pkg/util/common/common_test.go

```diff
@@ -127,6 +137,33 @@ func TestStructToMap(t *testing.T) {
 					},
 				},
 			},
-		},
-	}, StructToMap(top))
+			"mymap": map[string]interface{}{
+				"n1": map[string]interface{}{
+					"moreNested": []interface{}{
+						map[string]interface{}{
+							"id":          "toto",
+							"name":        "ms1",
+							"value":       float64(1),
+							"JSONLessStr": "42",
+						},
+						map[string]interface{}{
+							"name":        "ms2",
+							"value":       float64(2),
+							"JSONLessStr": "",
+						},
+					},
+				},
+			},
+		}},
+		{"sliceOfStructs", topSliceOfStructs, map[string]interface{}{}},
```

Comment:
You can keep spaces in the test case names for consistency - `t.Run` handles that.

Reply:
✔️